Client.xcodeproj/project.pbxproj

```diff
@@ -6558,7 +6471,11 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				INFOPLIST_FILE = UnitTests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
```

Comment:
XCode 11 wanted did this automatically, apprently the format for search paths is a bit different now